services/mailservice/mail_test.go

```diff
@@ -91,26 +93,26 @@ func TestMailConnectionAdvanced(t *testing.T) {
 }
 
 func TestSendMailUsingConfig(t *testing.T) {
-	cfg, _, _, err := LoadConfig("config.json")
+	cfg, _, _, err := utils.LoadConfig("config.json")
 	require.Nil(t, err)
-	T = GetUserTranslations("en")
+	utils.T = utils.GetUserTranslations("en")
 
 	var emailTo = "test@example.com"
 	var emailSubject = "Testing this email"
 	var emailBody = "This is a test from autobot"
 
 	//Delete all the messages before check the sample email
-	DeleteMailBox(emailTo)
+	utils.DeleteMailBox(emailTo)
```

Comment:
Can `utils/inbucket.go` be moved into this package as well and be marked as test code? I don't think it's used outside of testing this package

Reply:
Good point, I going to take a look into it.

Reply:
It is used in the tests of the `api4/team_test.go` and `api4/user_test.go`. We still can move it to a `mailservicetestutils` package or something like that inside the `mailservice` directory. and Import it from wherever is needed.

Reply:
Done! :)